var/spack/repos/builtin/packages/automake/package.py

```diff
@@ -23,6 +25,7 @@ class Automake(AutotoolsPackage, GNUMirrorPackage):
 
     depends_on('autoconf', type='build')
     depends_on('perl', type=('build', 'run'))
+    depends_on('gzip', type='build')
```

Comment:
But having a new maintainer is certainly welcome!

Reply:
Yes! But the reason I added `gzip` here is because otherwise, `automake` was failing to build on my alpine linux system--and this change fixes that! I believe it's the particular commands that automake runs when it builds itself. Please see the log I've attached in the PR description: https://github.com/spack/spack/pull/29230#issue-1150894498.